docs/software/container-development-guide.md

```diff
@@ -162,6 +162,15 @@ Images that have passed acceptance testing should be tagged as `stable`:
 
 1. Get the sha256 repo digest of the image that the user has tested.
    All you need is the part that starts with `sha256:...` (aka the `<DIGEST>`).
+   
+   A Kubernetes user can get the digest by from the "Image" and "Image ID" lines obtained by running:
+   
+         kubectl describe pod <POD>
+   
+   A Docker user can get the digest from running:
+   
+         docker image inspect <IMAGE NAME>:<TAG> | jq '.[0].Id'
```

Comment:
i take it only the `Id` in the first item in the list is relevant?

Reply:
The list only has one item. (Or zero if there's no match.) I don't get why it's a list -- I tried specifying just the image name without the tag -- maybe it'd give me all the images with that name or something -- but no dice.